README.md

```diff
@@ -2,10 +2,8 @@
 
 ### CocoaPods: The Cocoa dependency manager
 
-[![Build Status](http://img.shields.io/travis/CocoaPods/CocoaPods/master.svg?style=flat)](https://travis-ci.org/CocoaPods/CocoaPods)
-[![Gem Version](http://img.shields.io/gem/v/cocoapods.svg?style=flat)](http://badge.fury.io/rb/cocoapods)
-[![Maintainability](https://api.codeclimate.com/v1/badges/8f0fe544baf2ae1acc2b/maintainability)](https://codeclimate.com/github/CocoaPods/CocoaPods/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/8f0fe544baf2ae1acc2b/test_coverage)](https://codeclimate.com/github/CocoaPods/CocoaPods/test_coverage)
```

Comment:
did e lose codeclimate as part of this?

Reply:
Not as part of this - I assumed it wasn't working because I have never seen it in PR statuses.

Reply:
I thought we got it to work btw, is the link not going to be visible anymore?

Reply:
I looked at the link... frankly I'm not sure what I'm seeing.  
Looks like some sort of scoring system + another linter?  
Seems pretty hard to determine what version of the code it checked and it doesn't actually report anything to the PRs.

Who was in charge of it?

Reply:
I recently updated this, we don't check on PRs but the badge should be up to date

Reply:
It updates on merges to `master` I believe